templates/validator/tables.html

```diff
@@ -207,8 +205,7 @@
                           }
                         {{ end }}
                         return data
-                      },
-                    },
+                      }),
```

Comment:
I do not get this `Processing...`, the row for the next withdrawal is just missing.
This should not be an issue that appeared because of my changes, the current master also does not show the row.
If I remember correctly I did test it on a branch where it still worked and my changes looked fine but I can't find such a branch that starts anymore, maybe because of further db changes.

Reply:
Master branch:
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/109136188/1690e934-a563-408c-851d-60f5573625c2)
Your branch (stays like this):
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/109136188/527f4c6c-4df8-4a3c-b5fa-0e14fea6c07d)
Your branch after manuelly setting `NextWithdrwawalRow` to nil 
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/109136188/ba470ad9-5483-499f-9939-8d3d56e10c45)

Reply:
Thanks for the screenshots.
This should be fixed now. I could not test it on Sepolia so I used the manual entry.